dev-python/sipvicious/sipvicious-0.3.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..9} )
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1
+
+DESCRIPTION="SIPVicious is a SIP security package"
+HOMEPAGE="https://github.com/EnableSecurity/sipvicious/wiki"
+SRC_URI="https://github.com/EnableSecurity/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+BDEPEND=""
```

Comment:
If it really doesn't need anything, just skip it.

IUSE is also empty, remove it